hwilib/devices/trezor.py

```diff
@@ -754,6 +787,20 @@ def toggle_passphrase(self) -> bool:
                 print(PIN_MATRIX_DESCRIPTION, file=sys.stderr)
         return True
 
+    @trezor_exception
+    def can_sign_taproot(self) -> bool:
+        """
+        Trezor T supports Taproot in firmware versions greater than (not including) 2.4.2.
+        Trezor One supports Taproot in firmware versions greater than (not including) 1.10.3.
+
+        :returns: False, always.
+        """
+        self._prepare_device()
+        if self.client.features.model == "T":
+            return bool(self.client.version > (2, 4, 2))
+        return bool(self.client.version > (1, 10, 3))
```

Comment:
This is just semantics but this reads and feels much better:

```suggestion
            return bool(self.client.version >= (2, 4, 3))
        return bool(self.client.version >= (1, 10, 4))
```

Reply:
Done.

I wasn't sure what number the next versions are going to be, but if they are definitely 2.4.3 and 1.10.4, then this is fine.